services/brig/test/integration/API/User/Account.hs

```diff
@@ -141,9 +141,59 @@ tests _ at opts p b c ch g aws =
       testGroup
         "temporary customer extensions"
         [ test' aws p "domains blocked for registration" $ testDomainsBlockedForRegistration opts b
+        ],
+      testGroup
+        "re-send activation email"
+        [ test' aws p "put /users/:uid/email" $ testReSendActivationEmail b
         ]
     ]
 
+testReSendActivationEmail :: Brig -> Http ()
+testReSendActivationEmail brig = do
+  (_, teamOwner, emailOwner : otherTeamMember : _) <- createPopulatedBindingTeamWithNamesAndHandles brig 2
```

Comment:
People have told me I'm a java developer at heart for coming up with this name.  :)

("Unrelated" question: do you enjoy non-actionable comments in reviews, or should I avoid them?)

Reply:
Can you rewrite this test so that "letexpire" is only called once?  It's probably move expensive than the test put together by about an order of magnitude.